tests/falco/commands_test.go

```diff
@@ -54,7 +54,7 @@ func TestFalco_Cmd_Version(t *testing.T) {
 		assert.NoError(t, res.Err(), "%s", res.Stderr())
 		assert.Equal(t, res.ExitCode(), 0)
 		assert.Regexp(t, regexp.MustCompile(
-			`Falco version:[\s]+[0-9]+\.[0-9]+\.[0-9](\-[0-9]+\+[a-f0-9]+)?[\s]+`+
+			`Falco version:[\s]+[0-9]+\.[0-9]+\.[0-9](\-[a-z0-9]+)?(\+[a-f0-9]+)?[\s]+`+
 				`Libs version:[\s]+(([0-9]+\.[0-9]+\.[0-9](\-[a-z0-9]+)?)|([a-f0-9]+))[\s]+`+
 				`Plugin API:[\s]+[0-9]+\.[0-9]+\.[0-9][\s]+`+
```

Comment:
Plugins API is a subset of semver syntax, so it's acceptable to keep this regex.